src/controllers/tasks/registered.office.address.controller.ts

```diff
@@ -63,13 +63,13 @@ const formatAddress = (address: RegisteredOfficeAddress): RegisteredOfficeAddres
   const addressClone: RegisteredOfficeAddress = JSON.parse(JSON.stringify(address));
   return {
     careOf: formatTitleCase(addressClone.careOf),
+    poBox: formatTitleCase(addressClone.poBox),
```

Comment:
can you move the formatAddress function into the fomat.ts module? may have to rename the function if it clashes with existing formatAddress